pkgs/shells/zsh/zsh-fzf-tab/default.nix

```diff
@@ -81,11 +81,11 @@ in stdenv.mkDerivation rec {
 
   passthru.updateScript = nix-update-script { };
 
-  meta = with lib; {
+  meta = {
     homepage = "https://github.com/Aloxaf/fzf-tab";
     description = "Replace zsh's default completion selection menu with fzf!";
-    license = licenses.mit;
-    maintainers = with maintainers; [ vonfry ];
-    platforms = platforms.unix;
+    license = lib.licenses.mit;
+    maintainers = with lib.maintainers; [ vonfry ];
```

Comment:
Would you like to become a mainter and remove me from the list? I don't use this any more.

Reply:
I don't use this either(I use the Fish shell), I just noticed that it hadn't been updated even though a release had happened. I would be glad to remove you if you don't want to maintain the package though(and no hard feelings if you do! Packages get orphaned all the time, it's just how these things happen with large package repos)

Reply:
>  I would be glad to remove you if you don't want to maintain the package though(and no hard feelings if you do! 

If you don't want to be a maintainer, please keep me as one continuously.